src/api-reference/request/v4.endpoints.markdown

```diff
@@ -147,6 +154,12 @@ None
 
 ## <a name="request-resource-update"></a>Update the content of an existing Request
 
+Update of the the following fields is supported : policy, comment, startDate, startTime, endDate, endTime, expensePolicy, name, businessPurpose, mainDestination, travelAgency, custom fields.<br>
+Other fields will be ignored.<br>
+If providing an "id" field, as returned by GET requests, the value must match the requestUuid parameter.<br>
+This endpoint supports partial update. You may submit only the fields to update in the body, fields not present in the body will remain unchanged.<br>
```

Comment:
This update contradicts line 174 below- which one is correct?

Reply:
Updated in new commit